pkg/controller/backupbucket/actuator.go

```diff
@@ -43,12 +45,17 @@ func (a *actuator) InjectClient(client client.Client) error {
 }
 
 func (a *actuator) Reconcile(ctx context.Context, bb *extensionsv1alpha1.BackupBucket) error {
+	bbConfig, err := helper.GetBackupBucketConfig(bb)
```

Comment:
why create a separate API for bucket config? It'll require both config, `bucketconfig` as well as s3 secret exist. what about making `EnableEncryption` part of the s3 secret?

Reply:
This definitely makes sense, thanks. Added it to the secret and removed the API.